From b0355cc205543ffd33752874295139d57c4fbc3e Mon Sep 17 00:00:00 2001
From: Wenzong Fan <wenzong.fan@windriver.com>
Date: Tue, 26 Sep 2017 07:59:51 +0000
Subject: [PATCH] Subject: [PATCH] keyutils: use relative path for link

The absolute path of the symlink will be invalid
when populated in sysroot, so use relative path instead.

Upstream-Status: Pending

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
Signed-off-by: Wenzong Fan <wenzong.fan@windriver.com>
{rebased for 1.6]
Signed-off-by: Armin Kuster <akuster808@gmail.com>

Index: keyutils-1.6/Makefile
===================================================================
--- keyutils-1.6.orig/Makefile
+++ keyutils-1.6/Makefile
@@ -184,7 +184,7 @@ ifeq ($(NO_SOLIB),0)
 	$(INSTALL) -D $(LIBNAME) $(DESTDIR)$(LIBDIR)/$(LIBNAME)
 	$(LNS) $(LIBNAME) $(DESTDIR)$(LIBDIR)/$(SONAME)
 	mkdir -p $(DESTDIR)$(USRLIBDIR)
-	$(LNS) $(LIBDIR)/$(SONAME) $(DESTDIR)$(USRLIBDIR)/$(DEVELLIB)
+	$(LNS) $(SONAME) $(DESTDIR)$(USRLIBDIR)/$(DEVELLIB)
 	sed \
 	-e 's,@VERSION\@,$(VERSION),g' \
 	-e 's,@prefix\@,$(PREFIX),g' \
